Use the ISO date format in savegame and screenshot names

diff -r 93ca1fa1d558 src/lang/afrikaans.txt
--- a/src/lang/afrikaans.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/afrikaans.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1990,7 +1990,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Laai Speletjie
 STR_4002_SAVE                                                   :{BLACK}Spaar
 STR_4003_DELETE                                                 :{BLACK}Uitvee
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte{P "" s} vry
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Kan nie skyf lees nie
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Speletjie Spaar Misluk{}{STRING}
diff -r 93ca1fa1d558 src/lang/brazilian_portuguese.txt
--- a/src/lang/brazilian_portuguese.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/brazilian_portuguese.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1998,7 +1998,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Abrir Jogo
 STR_4002_SAVE                                                   :{BLACK}Salvar
 STR_4003_DELETE                                                 :{BLACK}Remover
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabytes livres
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Incapaz de ler unidade
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Falha ao Salvar Jogo{}{STRING}
diff -r 93ca1fa1d558 src/lang/bulgarian.txt
--- a/src/lang/bulgarian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/bulgarian.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1999,7 +1999,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Отваряне на игра
 STR_4002_SAVE                                                   :{BLACK}Запази
 STR_4003_DELETE                                                 :{BLACK}Изтрий
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} мегабайт{P "" а} свободно
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Не може да прочете диска
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Запазването на играта е неуспешно{}{STRING}
diff -r 93ca1fa1d558 src/lang/catalan.txt
--- a/src/lang/catalan.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/catalan.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1997,7 +1997,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Carrega un Joc
 STR_4002_SAVE                                                   :{BLACK}Desa
 STR_4003_DELETE                                                 :{BLACK}Esborra
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabytes lliures
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Impossible llegir la unitat de disc
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Desat del Joc erroni{}{STRING}
diff -r 93ca1fa1d558 src/lang/croatian.txt
--- a/src/lang/croatian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/croatian.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1969,7 +1969,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Učitaj igru
 STR_4002_SAVE                                                   :{BLACK}Spremi
 STR_4003_DELETE                                                 :{BLACK}Obriši
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabajt{P "" a a} slobod{P an na no}
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Ne mogu pročitati disk
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Spremanje igre nije uspjelo{}{STRING}
diff -r 93ca1fa1d558 src/lang/czech.txt
--- a/src/lang/czech.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/czech.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -2055,7 +2055,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Otevřít hru
 STR_4002_SAVE                                                   :{BLACK}Uložit
 STR_4003_DELETE                                                 :{BLACK}Smazat
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} MB voln{P ý é ých}
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Nemohu číst z jednotky
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Uložení hry selhalo{}{STRING}
diff -r 93ca1fa1d558 src/lang/danish.txt
--- a/src/lang/danish.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/danish.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1994,7 +1994,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Hent spil
 STR_4002_SAVE                                                   :{BLACK}Gem
 STR_4003_DELETE                                                 :{BLACK}Slet
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabytes fri
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Kan ikke læse drevet
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Fejl under gemning af spil{}{STRING}
diff -r 93ca1fa1d558 src/lang/dutch.txt
--- a/src/lang/dutch.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/dutch.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1998,7 +1998,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Spel Laden
 STR_4002_SAVE                                                   :{BLACK}Opslaan
 STR_4003_DELETE                                                 :{BLACK}Verwijderen
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabytes vrij
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Kan niet lezen van schijf
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Opslaan Spel Mislukt{}{STRING}
diff -r 93ca1fa1d558 src/lang/english.txt
--- a/src/lang/english.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/english.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1998,7 +1998,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Load Game
 STR_4002_SAVE                                                   :{BLACK}Save
 STR_4003_DELETE                                                 :{BLACK}Delete
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte{P "" s} free
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Unable to read drive
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Game Save Failed{}{STRING}
diff -r 93ca1fa1d558 src/lang/english_US.txt
--- a/src/lang/english_US.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/english_US.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1997,7 +1997,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Load Game
 STR_4002_SAVE                                                   :{BLACK}Save
 STR_4003_DELETE                                                 :{BLACK}Delete
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte{P "" s} free
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Unable to read drive
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Game Save Failed{}{STRING}
diff -r 93ca1fa1d558 src/lang/esperanto.txt
--- a/src/lang/esperanto.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/esperanto.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1796,7 +1796,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Ŝarĝu Ludon
 STR_4002_SAVE                                                   :{BLACK}Konservu
 STR_4003_DELETE                                                 :{BLACK}Forviŝu
-STR_4004                                                        :{COMPANY} je {DATE_LONG}
+STR_4004                                                        :{COMPANY} je {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabitoko{P "" j} libera{P "" j}
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Ne eblas legi diskon
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Ludkonservado Fiaskis{}{STRING}
diff -r 93ca1fa1d558 src/lang/estonian.txt
--- a/src/lang/estonian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/estonian.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -2045,7 +2045,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Laadi mäng
 STR_4002_SAVE                                                   :{BLACK}Salvesta
 STR_4003_DELETE                                                 :{BLACK}Kustuta
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabaiti{P "" i} vaba
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Ei suuda kettalt lugeda
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Mängu salvestamine nurjus{}{STRING}
diff -r 93ca1fa1d558 src/lang/finnish.txt
--- a/src/lang/finnish.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/finnish.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1951,7 +1951,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Lataa peli
 STR_4002_SAVE                                                   :{BLACK}Tallenna
 STR_4003_DELETE                                                 :{BLACK}Poista
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megatavu{P "" a} vapaana
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Asemaa ei voi lukea.
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Tallennus epäonnistui.{}{STRING}
diff -r 93ca1fa1d558 src/lang/french.txt
--- a/src/lang/french.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/french.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1999,7 +1999,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Charger une partie
 STR_4002_SAVE                                                   :{BLACK}Sauvegarder
 STR_4003_DELETE                                                 :{BLACK}Supprimer
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} Mo libre{P "" s}
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Impossible d'accéder au disque
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}La sauvegarde à échoué{}{STRING}
diff -r 93ca1fa1d558 src/lang/galician.txt
--- a/src/lang/galician.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/galician.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1999,7 +1999,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Cargar Partida
 STR_4002_SAVE                                                   :{BLACK}Grabar
 STR_4003_DELETE                                                 :{BLACK}Borrar
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte{P "" s} libre{P "" s}
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}No se pode lee-la unidade
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Grabación da partida fallida{}{STRING}
diff -r 93ca1fa1d558 src/lang/german.txt
--- a/src/lang/german.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/german.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1998,7 +1998,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Spiel laden
 STR_4002_SAVE                                                   :{BLACK}Speichern
 STR_4003_DELETE                                                 :{BLACK}Löschen
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} Megabyte{P "" s} frei
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Laufwerk nicht betriebsbereit
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Spielspeicherung fehlgeschlagen{}{STRING}
diff -r 93ca1fa1d558 src/lang/hungarian.txt
--- a/src/lang/hungarian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/hungarian.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -2059,7 +2059,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Játék betöltése
 STR_4002_SAVE                                                   :{BLACK}Ment
 STR_4003_DELETE                                                 :{BLACK}Töröl
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabájt szabad
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Nem olvasható a meghajtó
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}A mentés nem sikerült{}{STRING}
diff -r 93ca1fa1d558 src/lang/icelandic.txt
--- a/src/lang/icelandic.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/icelandic.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1942,7 +1942,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Opna leik
 STR_4002_SAVE                                                   :{BLACK}Vista
 STR_4003_DELETE                                                 :{BLACK}Eyða
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabæti laus
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Get ekki lesið af drifi
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Vistun leiks mistókst{}{STRING}
diff -r 93ca1fa1d558 src/lang/italian.txt
--- a/src/lang/italian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/italian.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -2000,7 +2000,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Carica partita
 STR_4002_SAVE                                                   :{BLACK}Salva
 STR_4003_DELETE                                                 :{BLACK}Elimina
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte liber{P o i}
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Impossibile leggere dall'unità
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Salvataggio fallito{}{STRING}
diff -r 93ca1fa1d558 src/lang/japanese.txt
--- a/src/lang/japanese.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/japanese.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1913,7 +1913,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}ゲームを開く
 STR_4002_SAVE                                                   :{BLACK}保存
 STR_4003_DELETE                                                 :{BLACK}削除
-STR_4004                                                        :{COMPANY}、{DATE_LONG}
+STR_4004                                                        :{COMPANY}、{ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} MB 使用可能
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}ドライブが読み込めません
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}ゲームの保存が失敗しました{}{STRING}
diff -r 93ca1fa1d558 src/lang/korean.txt
--- a/src/lang/korean.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/korean.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1997,7 +1997,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}게임 불러오기
 STR_4002_SAVE                                                   :{BLACK}저장
 STR_4003_DELETE                                                 :{BLACK}파일 삭제
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} Mb 남음
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}드라이브를 읽을 수 없음
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}게임 저장 실패{}{STRING}
diff -r 93ca1fa1d558 src/lang/lithuanian.txt
--- a/src/lang/lithuanian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/lithuanian.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1840,7 +1840,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Atverti zaidima
 STR_4002_SAVE                                                   :{BLACK}Issaugoti
 STR_4003_DELETE                                                 :{BLACK}Trinti
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} Mb laisva
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Negali pasiekti disko
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Zaidimo issaugoti nepavyko{}{STRING}
diff -r 93ca1fa1d558 src/lang/norwegian_bokmal.txt
--- a/src/lang/norwegian_bokmal.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/norwegian_bokmal.txt	Fri Nov 21 14:56:46 2008 +0100
@@ -1995,7 +1995,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Last inn spill
 STR_4002_SAVE                                                   :{BLACK}Lagre
 STR_4003_DELETE                                                 :{BLACK}Slett
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte ledig
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Kan ikke lese fra disk
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Lagring av spillet mislyktes{}{STRING}
diff -r 93ca1fa1d558 src/lang/norwegian_nynorsk.txt
--- a/src/lang/norwegian_nynorsk.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/norwegian_nynorsk.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1903,7 +1903,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Last inn spel
 STR_4002_SAVE                                                   :{BLACK}Lagre
 STR_4003_DELETE                                                 :{BLACK}Slett
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte ledig
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Kan ikkje lese frå disk
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Lagring av spelet mislukkast{}{STRING}
diff -r 93ca1fa1d558 src/lang/piglatin.txt
--- a/src/lang/piglatin.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/piglatin.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1927,7 +1927,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Oadlay Amegay
 STR_4002_SAVE                                                   :{BLACK}Avesay
 STR_4003_DELETE                                                 :{BLACK}Eleteday
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} egabyte{P "" s}may eefray
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Unableway otay eadray ivedray
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Amegay Avesay Ailedfay{}{STRING}
diff -r 93ca1fa1d558 src/lang/polish.txt
--- a/src/lang/polish.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/polish.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -2065,7 +2065,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Wczytaj grę
 STR_4002_SAVE                                                   :{BLACK}Zapisz
 STR_4003_DELETE                                                 :{BLACK}Usuń
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} MB wolnego miejsca
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Nie można odczytać napędu
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Nie można zapisać gry{}{STRING}
diff -r 93ca1fa1d558 src/lang/portuguese.txt
--- a/src/lang/portuguese.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/portuguese.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1958,7 +1958,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Abrir Jogo
 STR_4002_SAVE                                                   :{BLACK}Guardar
 STR_4003_DELETE                                                 :{BLACK}Eliminar
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabytes livres
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Não é possível ler unidade
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Falha ao Guardar Jogo{}{STRING}
diff -r 93ca1fa1d558 src/lang/romanian.txt
--- a/src/lang/romanian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/romanian.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1983,7 +1983,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Încarcă joc
 STR_4002_SAVE                                                   :{BLACK}Salvează
 STR_4003_DELETE                                                 :{BLACK}Şterge
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte{P "" s} liberi
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Discul nu a putut fi citit
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Salvarea jocului eşuată{}{STRING}
diff -r 93ca1fa1d558 src/lang/russian.txt
--- a/src/lang/russian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/russian.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1992,7 +1992,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Загрузить игру
 STR_4002_SAVE                                                   :{BLACK}Сохранить
 STR_4003_DELETE                                                 :{BLACK}Удалить
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} Мб свободно
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Не удалось прочитать диск
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Невозможно сохранить игру{}{STRING}
diff -r 93ca1fa1d558 src/lang/simplified_chinese.txt
--- a/src/lang/simplified_chinese.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/simplified_chinese.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1995,7 +1995,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}读取存档
 STR_4002_SAVE                                                   :{BLACK}保存
 STR_4003_DELETE                                                 :{BLACK}删除
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte(s) 可用
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}不能读取驱动器
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}保存游戏失败{}{STRING}
diff -r 93ca1fa1d558 src/lang/slovak.txt
--- a/src/lang/slovak.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/slovak.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -2062,7 +2062,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Nahrat hru
 STR_4002_SAVE                                                   :{BLACK}Ulozit
 STR_4003_DELETE                                                 :{BLACK}Vymazat
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabajtov volnych
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Zariadenie je necitatelne
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Ulozenie hry zlyhalo{}{STRING}
diff -r 93ca1fa1d558 src/lang/slovenian.txt
--- a/src/lang/slovenian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/slovenian.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -2038,7 +2038,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Naloži igro
 STR_4002_SAVE                                                   :{BLACK}Shrani
 STR_4003_DELETE                                                 :{BLACK}Izbriši
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabajt{P "" a i ov} prosto
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Nemogoče branje pogona
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Shranjevanje igre ni uspelo{}{STRING}
diff -r 93ca1fa1d558 src/lang/spanish.txt
--- a/src/lang/spanish.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/spanish.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1998,7 +1998,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Cargar Juego
 STR_4002_SAVE                                                   :{BLACK}Guardar
 STR_4003_DELETE                                                 :{BLACK}Borrar
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabytes libres
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}No es posible leer la unidad
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Error guardando juego{}{STRING}
diff -r 93ca1fa1d558 src/lang/swedish.txt
--- a/src/lang/swedish.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/swedish.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1995,7 +1995,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Ladda spel
 STR_4002_SAVE                                                   :{BLACK}Spara
 STR_4003_DELETE                                                 :{BLACK}Ta bort
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte{P "" s} ledigt
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Kan inte läsa från disk
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Sparandet av spelet misslyckades{}{STRING}
diff -r 93ca1fa1d558 src/lang/traditional_chinese.txt
--- a/src/lang/traditional_chinese.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/traditional_chinese.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1995,7 +1995,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}載入遊戲
 STR_4002_SAVE                                                   :{BLACK}儲存
 STR_4003_DELETE                                                 :{BLACK}刪除
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}空間尚有 {COMMA} MB
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}無法讀取磁碟
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}遊戲儲存失敗{}{STRING}
diff -r 93ca1fa1d558 src/lang/turkish.txt
--- a/src/lang/turkish.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/turkish.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1935,7 +1935,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Yükle
 STR_4002_SAVE                                                   :{BLACK}Kaydet
 STR_4003_DELETE                                                 :{BLACK}Sil
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} MB bos
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Sürücü okunamıyor
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Oyun kaydedilemedi{}{STRING}
diff -r 93ca1fa1d558 src/lang/ukrainian.txt
--- a/src/lang/ukrainian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/ukrainian.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -2120,7 +2120,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Завантажити гру
 STR_4002_SAVE                                                   :{BLACK}Зберегти
 STR_4003_DELETE                                                 :{BLACK}Видалити
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} мегабайт{P "" и ""}  вільно
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Неможливо прочитати пристрій
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Помилка запису гри{}{STRING}
diff -r 93ca1fa1d558 src/lang/unfinished/frisian.txt
--- a/src/lang/unfinished/frisian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/unfinished/frisian.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1036,7 +1036,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Spul Lade
 STR_4002_SAVE                                                   :{BLACK}Opslaan
 STR_4003_DELETE                                                 :{BLACK}Fuortsmite
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Koe net fan 'e skiif lêze
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Opslaan Mislearre{}{STRING}
 STR_4008_UNABLE_TO_DELETE_FILE                                  :{WHITE}Kin triem net fuortsmite
diff -r 93ca1fa1d558 src/lang/unfinished/greek.txt
--- a/src/lang/unfinished/greek.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/unfinished/greek.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1530,7 +1530,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Φόρτωση Παιχνιδιού
 STR_4002_SAVE                                                   :{BLACK}Αποθήκευση
 STR_4003_DELETE                                                 :{BLACK}Διαγραφή
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megabyte{P "" s} ελεύθερα
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Αδύνατη η ανάγνωση του δίσκου
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Αποθήκευση Παιχνιδιού Απέτυχε{}{STRING}
diff -r 93ca1fa1d558 src/lang/unfinished/latvian.txt
--- a/src/lang/unfinished/latvian.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/unfinished/latvian.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1620,7 +1620,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Ielādēt spēli
 STR_4002_SAVE                                                   :{BLACK}Saglabāt
 STR_4003_DELETE                                                 :{BLACK}Dzēst
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Nevar lasīt disku
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Spēles saglabāšana neizdevās{}{STRING}
 STR_4008_UNABLE_TO_DELETE_FILE                                  :{WHITE}Nevar dzēst failu
diff -r 93ca1fa1d558 src/lang/welsh.txt
--- a/src/lang/welsh.txt	Fri Nov 21 14:47:31 2008 +0100
+++ b/src/lang/welsh.txt	Fri Nov 21 14:56:47 2008 +0100
@@ -1995,7 +1995,7 @@ STR_4001_LOAD_GAME                      
 STR_4001_LOAD_GAME                                              :{WHITE}Llwytho Gêm
 STR_4002_SAVE                                                   :{BLACK}Cadw
 STR_4003_DELETE                                                 :{BLACK}Dileu
-STR_4004                                                        :{COMPANY}, {DATE_LONG}
+STR_4004                                                        :{COMPANY}, {ISO_DATE}
 STR_4005_BYTES_FREE                                             :{BLACK}{COMMA} megadid yn rhydd
 STR_4006_UNABLE_TO_READ_DRIVE                                   :{BLACK}Methu darllen y gyriant
 STR_4007_GAME_SAVE_FAILED                                       :{WHITE}Methwyd â Chadw Gêm{}{STRING}
